W-18366225-custom-metrics-feedback-duke (#386)
mulesoft/docs-monitoring: modules/ROOT/pages/anypoint-custom-metrics-connector.adoc

You must also xref:am-installing.adoc[install the Anypoint Monitoring Agent].

> Review xref:connectors::metrics/custom-metrics-connector.adoc[Custom Metrics Connector] to understand connector configuration details. Connector configuration is required in your Mule app flow before you deploy or redeploy the app the application to a server or server cluster. 

Author: ```suggestion
Review xref:connectors::metrics/custom-metrics-connector.adoc[Custom Metrics Connector] to understand connector configuration details. Connector configuration is required in your Mule app flow before you deploy or redeploy your app to a server or server cluster. 
```

. From Anypoint Monitoring:
.. Click *Custom Metrics*.
.. Click *Custom Metrics Connector* for the latest Mule version to open the connector on Anypoint Exchange.
.. Click *Dependency Snippet*, and copy the Maven snippet.
. In Anypoint Studio, add the `<dependency/>` snippet for the latest version of the connector to your Mule project's `pom.xml` file within the `<dependencies/>` element, for example:
+
----
<dependencies>
...
  <dependency>
    <groupId>com.mule.modules</groupId>
    <artifactId>mule-custom-metrics-extension</artifactId>
    <version>2.3.1</version>
    <classifier>mule-plugin</classifier>
  </dependency>
</dependencies>
-----
+

> The connector is now available in the Studio palette. 

Author: ```suggestion
Your connector version value might differ from the example. After you add the dependency, the connector becomes available in the Studio palette. 
```

+

> If you encounter issues with this step, see xref:studio::add-modules-in-studio-to.adoc[] to add the connector from Exchange through the Studio pallet.

Reviewer: ```suggestion
If you encounter issues with this step, see xref:studio::add-modules-in-studio-to.adoc[] to add the connector in Studio from Exchange.
```

Reviewer: I'm just not sure "Studio pallet" is right. Maybe it is! You can ignore if it's fine.
